appcues/src/main/java/com/appcues/debugger/ui/CaptureConfirmDialog.kt

```diff
@@ -0,0 +1,220 @@
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.composed
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.geometry.toRect
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.graphics.drawscope.DrawScope
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.Role
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import androidx.compose.ui.window.Dialog
+import coil.compose.rememberAsyncImagePainter
+import com.appcues.R.string
+import com.appcues.debugger.DebuggerViewModel
+import com.appcues.debugger.screencapture.Capture
+import com.appcues.ui.theme.AppcuesColors
+
+@Composable
+internal fun CaptureConfirmDialog(capture: Capture, debuggerState: MutableDebuggerState, debuggerViewModel: DebuggerViewModel) {
+    val text = remember { mutableStateOf(capture.displayName) }
+
+    // don't show if current debugger is paused
+    if (debuggerState.isPaused.value) return
+
+    Dialog(
+        onDismissRequest = {
+            debuggerViewModel.closeExpandedView()
+        },
+    ) {
+        Column(
+            modifier = Modifier
+                .background(AppcuesColors.DebuggerBackground, RoundedCornerShape(6.dp))
+                .padding(9.dp)
+                .verticalScroll(rememberScrollState())
+        ) {
+            Header(debuggerViewModel = debuggerViewModel)
+            CaptureContents(debuggerViewModel = debuggerViewModel, capture = capture, text = text)
+        }
+    }
+}
+
+@Composable
+private fun Header(debuggerViewModel: DebuggerViewModel) {
+    Row(verticalAlignment = Alignment.CenterVertically) {
+        Text(
+            modifier = Modifier.padding(start = 16.dp),
+            text = stringResource(id = string.appcues_screen_capture_title),
+            style = TextStyle(
+                fontSize = 20.sp
+            )
+        )
+        Spacer(modifier = Modifier.weight(1.0f))
+        Spacer(
+            modifier = Modifier
+                .size(48.dp, 48.dp)
+                .clickable(
+                    onClick = { debuggerViewModel.closeExpandedView() },
+                    role = Role.Button,
+                    interactionSource = remember { MutableInteractionSource() },
+                    indication = rememberRipple(bounded = false, 24.dp),
+                    onClickLabel = stringResource(id = string.appcues_screen_capture_dismiss)
+                )
+                .drawBehind {
+                    xShapePath(Color.Black, 16.dp)
+                        .also { drawPath(path = it, Color.Black) }
```

Comment:
wondering is Color.Black will work for Dark mode

Reply:
also this looks as though this was copied from the skippable trait, I guess we could  define the xShapePath somewhere we can share in other places.

Reply:
yes on both points - I'll see if I can find a shared spot for that xShapePath, good call!

Dark mode - I did check this, in short, it's not handled at all throughout the debugger currently. I figured I would leave that be and we'd need to do a follow up pass to handle dark mode if desired.

Reply:
Dark mode: good point, I will create a task for it